CodeJam.Main/Strings/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+
+using JetBrains.Annotations;
+
+namespace CodeJam.Strings
+{
+	/// <summary>
+	/// <see cref="StringBuilder"/> class extensions.
+	/// </summary>
+	[PublicAPI]
+	public class StringBuilderExtensions
+	{
+		/// <summary>
+		/// Determines whether the StringBuilder is null or empty.
+		/// </summary>
+		/// <param name="value">The StringBuilder to test.</param>
+		/// <returns>
+		/// <c>true</c> if <paramref name="value"/> is empty; otherwise, <c>false</c>.
+		/// </returns>
+		[Pure]
+		public static bool IsNullOrEmpty([CanBwNull] this StringBuilder value)
```

Comment:
NotNullNorEmpty will be a good addition. otherwise, all is fine

Reply:
Changed to IsNullOrEmpty.
Do you suggest having two methods ?

Reply:
Done